requirements/development.txt

```diff
@@ -10,6 +10,8 @@
     # via
     #   -r requirements/base.in
     #   -r requirements/development.in
+Authlib==1.2.0
```

Comment:
These need to be added in `development.in`. After adding there, this file should be rebuilt according to the steps in `CONTRIBUTING.md`

Reply:
@craig-rueda Thanks for the tip, I've done all the steps!